ODIN_II/SRC/multipliers.cpp

```diff
@@ -491,18 +491,21 @@ void define_mult_function(nnode_t* node, FILE* out) {
 
     for (i = 0; i < node->num_input_pins; i++) {
         if (i < node->input_port_sizes[flip ? 1 : 0]) {
-            npin_t* driver_pin = flip
-                                     ? node->input_pins[i + node->input_port_sizes[0]]->net->driver_pin
-                                     : node->input_pins[i]->net->driver_pin;
+            oassert((flip ? node->input_pins[i + node->input_port_sizes[0]]->net->num_driver_pins : node->input_pins[i]->net->num_driver_pins) == 1);
```

Comment:
store the index in a variable, otherwise, the ternary is unreadable ; especially if done twice